### DISCUSSION TITLE
Searching for jumulus for macOS Catalina, new software can't run

### DISCUSSION THREAD

**Discussion by hirscherich** [jamulussoftware/jamulus#3222] [Technical Support & Help] [answered]
Hi everybody, for an older MacBook Retina w/ version MacOs Catalina i'm searching a running jumulus version. W/ the newest one my computer don't run. Who can help me, i need an older version.

Thanks for your efforts.

Many greetings
hirsch

**Comment by hirscherich** [comment]
I've found an older version 3.9.

**Reply by voxxman** [reply]
Hi Hirscherich,
I've just upgraded my Mac to try and solve another issue and have found that Jamulus is not working for me either.  I can only get to Catalina, no further.  Did you manage to find a version and install a version of Jamulus that works.  I have downloaded 3.9, but Apple is complaining that the DMG can't be verified, so won't install.

Thanks for any info you may have.
Kind regards
Rich

**Reply by ann0see** [reply]
Yes. It is an unsigned version. You can get the latest legacy version from the page I linked above. To circumvent the non signed error, right click in the applications folder on Jamulus and then tap open in the context menu. You might need to donthat two times.

**Reply by voxxman** [reply]
Cheers Ann0see, much appreciated.  I'll give that a go later

**Reply by voxxman** [reply]
Cheers Ann0see, that worked, thank you so much

**Reply by ann0see** [reply]
Depending on your Mac's specs, you can update it to a higher OS via OpenCore legacy patcher. If you use it as primary device, I'd consider looking for alternatives. The M series chips seem to be a big step forward.

**Comment by ann0see** [comment, marked as answer]
Does the legacy version work? (Check the link to old (legacy) versions on https://jamulus.io/wiki/Installation-for-Macintosh - see the Note there). We no longer officially support it but it's still being built.
